app/assets/stylesheets/forms.scss

```diff
@@ -34,3 +34,8 @@ button {
   @extend .btn;
   @extend .btn-default;
 }
+
+#new_mainstream_browse_page_archival_form,
```

Comment:
Worth double checking in the relevant styleguide, but I think css selectors are hyphen separated.